tests/regression/tests/REQUEST-933-APPLICATION-ATTACK-PHP/933160.yaml

```diff
@@ -465,6 +503,57 @@ tests:
               Accept: text/xml,application/xml,application/xhtml+xml,text/html;q=0.9,text/plain;q=0.8,image/png,*/*;q=0.5
             method: POST
             port: 80
-            uri: /
+            uri: /post
+          output:
+            log_contains: id "933160"
+  - test_title: 933160-30
+    desc: |
+      PHP string function call
+      payload: return"system"(xyz).s
+    stages:
+      - stage:
+          input:
+            dest_addr: 127.0.0.1
+            headers:
+              Host: localhost
+              User-Agent: "OWASP CRS test agent"
+              Accept: text/xml,application/xml,application/xhtml+xml,text/html;q=0.9,text/plain;q=0.8,image/png,*/*;q=0.5
+            method: GET
+            port: 80
+            uri: /get?foo=return%22system%22%28xyz%29.s
           output:
             log_contains: id "933160"
+  - test_title: 933160-31
+    desc: |
+      False Positive test, missing mandatory parenthesis
+      payload: /eval(
```

Comment:
In this case, I changed the payload to eval because `strrev` has been moved. I think that the key point here is running an FP test checking the missing parenthesis, not the matched string itself.